tvx/drivers/dummy_testingt.go

```diff
@@ -0,0 +1,28 @@
+package drivers
+
+import (
+	"fmt"
+	"os"
+)
+
+var t dummyTestingT
+
+type dummyTestingT struct{}
```

Comment:
can we remove this? it seems like a cludge - panic directly where this would panic, rather than making the code look like it's been imported from a test

Reply:
There are probably >100 places that use an interface subset of testing.T (all the `require` and `assert` statements). I'd rather not change all that right now, as that would make the PR twice as big.

I know this is ugly, but I didn't think of a better way, and updating all the code to get rid of `testify` framework doesn't seem like a good use of our time right now.

Reply:
just flagging that this is a lot of technical debt we're taking on at the start of this and now are gonna have to be maintaining :-|